### ISSUE TITLE
Accessibility Review

### ISSUE THREAD

**Issue by austinlparker** [open-telemetry/opentelemetry.io#190]
Per https://github.com/open-telemetry/community/issues/370#issuecomment-641370166 and other D&I measures, we'll need to do some work on making the website ARIA friendly and ensuring that we're using best practices for accessibility. This issue will hold the review from myself and others on figuring out what needs to be changed.

**Comment by austinlparker** [comment]
Hopefully the new theme helps with this, but I'd love if someone more read up on web accessibility could take a look at the site and let us know what we're missing!

**Comment by pankajch82** [comment]
Hi @austinlparker, 
Few open issues : 

- Provide alternative text for images
-  Ensure links or controls that open new windows or frames do not open without a warning
- Ensure link text is meaningful within the context

**Comment by pankajch82** [comment]
Here are some details for home page : 

- Provide alternative text for images
 <img src="/img/logos/opentelemetry-stacked-color.png" class="card-img-top mx-auto">-

- Ensure links or controls that open new windows or frames do not open without a warning
<a target="_blank" href="https://medium.com/opentelemetry" class="nav-link"><span>Blog</span></a>

- Ensure link text is meaningful within context
It is important for users to be able to discern the purpose of all links. Meaningful link text should not be overly general and should clearly describe the content to be found or action to be performed by the link. For example, do not use generic text such as "click here", "read more"
"Read More" has been used on the home page.

I can take a look at other pages too if you think this is helpful.

**Comment by harshita19244** [comment]
I can suggest some UI fixes for the same and additionally submit a pr. Could you assign me the issue?

**Comment by austinlparker** [comment]
Sure thing.

**Comment by harshita19244** [comment]
@austinlparker Sorry but could you guide me as to where the code corresponding to the navbar on the homepage is? I cant seem to find it.

**Comment by austinlparker** [comment]
The navigation bar is generated automatically from the content directory - if you look at the `_index.md` files under `content/en`, the front matter specifies which get displayed in the nav bar.

There's more info here: https://www.docsy.dev/docs/language/

**Comment by harshita19244** [comment]
![Screenshot (11)](https://user-images.githubusercontent.com/60603854/113470246-babb8980-9471-11eb-9da8-875049e639a4.png)
Been trying to add icons in the navbar for improved accessibility, but the changes do not show up on make serve. @austinlparker Am I missing something? I followed the documentation of docsy, and their procedure of adding navbar icons, but it does not seem to work out.

**Comment by harshita19244** [comment]
https://github.com/open-telemetry/opentelemetry.io/pull/509
Kindly review the above pr. It solves a number of accessibility issues. (solves #190 and #497)I have added alt text for all images and resolved dead links on the website. Let me know if these work out, and if they can be merged to the main branch.

**Comment by sirandysaunders** [comment]
Yes, Agreed! Can you continue to assist me? You guys are far better at this than I.

**Comment by sirandysaunders** [comment]
Please 🙏
